cmd/publicize/main.go

```diff
@@ -0,0 +1,189 @@
+package main
+
+import (
+	"errors"
+	"flag"
+	"fmt"
+	"io/ioutil"
+	"os"
+	"strings"
+	"time"
+
+	"github.com/ghodss/yaml"
+	"github.com/sirupsen/logrus"
+
+	kerrors "k8s.io/apimachinery/pkg/util/errors"
+	"k8s.io/apimachinery/pkg/util/sets"
+
+	"k8s.io/test-infra/prow/githubeventserver"
+	"k8s.io/test-infra/prow/interrupts"
+	"k8s.io/test-infra/prow/logrusutil"
+
+	"k8s.io/test-infra/pkg/flagutil"
+	"k8s.io/test-infra/prow/config/secret"
+	prowflagutil "k8s.io/test-infra/prow/flagutil"
+)
+
+type Config struct {
+	Bonds []Bond `json:"bonds,omitempty"`
+}
+
+func (c *Config) validate() error {
+	var errs []error
+	for index, bond := range c.Bonds {
+		if err := bond.validate(); err != nil {
+			errs = append(errs, fmt.Errorf("bond[%d]: %w", index, err))
+		}
+	}
```

Comment:
Would it make sense to validate for conficts (one repo appearing in multiple `from`s or multiple `to`s. Would that be a supported case?

Reply:
Yea, that sounds good to me. I will add that validation as well.

Reply:
This is outstanding, no?

Reply:
Yes. I wanted first to address the changes in the configuration file, then extend the validation code.